### PR TITLE
KIB to DKP compatibility matrix

### PR DESCRIPTION
**Pull request by mmcelroy91**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-92208

## Description of changes being made
Take KIB compatibility table from Confluence https://d2iq.atlassian.net/wiki/spaces/DENT/pages/29918909/Konvoy+Image+Builder#Compatible-versions and add through PR to 2.2
### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4603.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/2.2/image-builder/index.md [hunk 1]
```diff
@@ -9,3 +9,40 @@ menuWeight: 70
...
+
+## Prerequisites
+
+Before you begin, you must ensure your versions of KIB and DKP are compatible:
+
+-  Download the supported version of the KIB bundle from the KIB Version section of the chart below (prefixed with konvoy-image-bundle) for your Operating System.
+### Supported versions
+
+
+| Kubernetes Support  | Version |
```
Comment (cmastr) [inline pages/dkp/konvoy/2.2/image-builder/index.md:23]: I don't know that these are the correct Kubernetes versions for DKP 2.2.

File: pages/dkp/konvoy/2.2/image-builder/index.md [hunk 2]
```diff
@@ -9,3 +9,40 @@ menuWeight: 70
...
+
+| Kubernetes Support  | Version | 
+|------------|----------------------|
+| Minimum | 1.22.0 |
+| Maximum | 1.23.x |
+| Default | 1.22.0 |
+| EKS Default | 1.22.x |
+
+
+DKP 2.3 comes with support for Kubernetes 1.23, enabling you to benefit from the latest features and security fixes in upstream Kubernetes. This release comes with approximately 47 enhancements.
```
Comment (cmastr) [inline pages/dkp/konvoy/2.2/image-builder/index.md:31]:
```suggestion
```

File: pages/dkp/konvoy/2.2/image-builder/index.md [hunk 3]
```diff
@@ -9,3 +9,40 @@ menuWeight: 70
...
+-  Create a working `docker` setup.
+
+## Compatible versions
+Along with the KIB Bundle, we publish a file containing checksums for the bundle files.  The recommendation for using these checksums is to verify the integrity of the downloads.
+
+
+| DKP Version  | KIB Version | 
+|------------|----------------------|
+| v2.3.0 | v1.19.9 |
```
Comment (cmastr) [inline pages/dkp/konvoy/2.2/image-builder/index.md:40]: Don't need 2.3 here

**Review by cmastr — COMMENTED** [review]
need to remove some of the 2.3 versions.

File: pages/dkp/konvoy/2.2/image-builder/index.md (hunk 1 above)
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/image-builder/index.md:23]: You know what, I don't either! HA!  Will check with Eng

Comment (cmastr) [reply pages/dkp/konvoy/2.2/image-builder/index.md:23]: This should already be in the release notes for 2.2

Comment (cneth) [reply pages/dkp/konvoy/2.2/image-builder/index.md:23]: I don't really see the point in having the Kubernetes version stuff  on this page at all, honestly.    It's not on the 2.3 page.  Let's just nuke it.

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/2.2/image-builder/index.md (hunk 1 above)
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/image-builder/index.md:23]: Ok, and just will just keep the KIB version matrix.

**Review by dkoshkin — APPROVED** [review]

**Pull request merged**